tests/simulations/ringtest/reference/cellstate_1.json

```diff
@@ -1,7 +1,12 @@
 {
   "B_BallStick": {
     "gid": 1.0,
-    "nSecAll": 3,
+    "nSecAll": 3.0,
```

Comment:
do we need nSecAll if it is just the sum of the others? I am wondering because if this number may be extrapolated from the others maybe we should remove it completely. Otherwise we could have wierd bugs down the line

Reply:
It is the sum of the others, and it is needed for reporting https://github.com/openbraininstitute/neurodamus/blob/87afb1ada216bedd1d6ce48393aa6dae5b9a27d3/neurodamus/target_manager.py#L546
The value is computed inside emodel hoc template, e.g. `B_BallStick.hoc`

Reply:
can't we calculate it from the other sections in the .hoc file?

Reply:
`nSecAll` is provided in all the BBP hoc templates, why can't use it?

Reply:
I was not aware of that. If it is customary, ok